optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-api/src/main/java/org/optaplanner/workbench/screens/solver/model/ScoreDirectorFactoryConfigModel.java

```diff
@@ -24,6 +24,8 @@
 
     protected ScoreDefinitionTypeModel scoreDefinitionType = null;
 
+    protected String ksessionName = null;
+
     protected List<String> scoreDrlList = null;
```

Comment:
Can scoreDrlList be removed entirely? We shouldn't support that at all in wb I believe.